npm/builder.js

```diff
@@ -27,10 +27,16 @@ function Builder () {
     'node_modules/opal-runtime/src/opal.js',
     'build/asciidoctor-lib.js'
   ];
-  if (process.env.ASCIIDOCTOR_CORE_VERSION) {
-    this.asciidoctorCoreVersion = process.env.ASCIIDOCTOR_CORE_VERSION;
+  var asciidoctorRef = process.env.ASCIIDOCTOR_CORE_VERSION;
+  if (asciidoctorRef && ~asciidoctorRef.indexOf('/')) {
```

Comment:
In my opinion, the tilde operator is uncommon, could you use `includes` instead ? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/includes